posts/01-first-map/byandell/uwmadison.md

```diff
@@ -0,0 +1,9 @@
+# UW-Madison
+
+[UW-Madison](https://wisc.edu) is located in Madison, WI.
+There are 40k+ students and thousands of faculty and staff.
+Here is the
+[campus map](https://map.wisc.edu)
+and below is an interactive image of the campus area.
+
+<embed type="text/html" src="osm.html" width="600" height="600">
```

Comment:
The markdown looks good but I would update the src path from "osm.html" to "uwmadison.html"

Reply:
Done. Thanks for catching this.